### DISCUSSION TITLE
Ender-3 S1 Pro STM32F401 with CR4NS200141C13 board

### DISCUSSION THREAD

**Discussion by IamBelka** [mriscoc/Ender3V2S1#745] [Q&A] [answered]
Hi, there is a stock printer with an STM32F401 chip and a CR4NS200141C13 board. Default Firmware Ender-3 S1_Pro_HWv24S1_301_SWV2.0.8.26F4_FDM_LASER 02 Feb. 2023.
I'm trying to flash Experimental-Ender3S1-F4-LA-20221222.bin
but the printer does not start. 
What could be wrong?

**Comment by fuzelet** [comment]
Based on this commit, you have an Ender 5 S1, not an Ender 3 S1 pro.

Creality CR4NS200141C13 as found in Ender-5 S1 With STM32F401RET6

https://github.com/MarlinFirmware/Marlin/pull/25382/commits/d36620b30ecf8a228b7bf74722aaa0e1a244819e

**Reply by IamBelka** [reply]
In appearance and according to the accompanying documentation, I have Ender 3 S1 pro, but with a strange board.

**Reply by icem4n** [reply]
I have the same:  Ender 3 S1 Pro I got in December with the CR4NS200141C13 board.  I haven't been able to find any compatible firmware.  Very frustrating.

**Reply by mriscoc** [reply]
Ask to Creality for pointing you where the firmware was posted, the filename of the firmware could guide us about what board is compatible with that printer.

**Reply by charliechan84** [reply]
as belka stated, i just purchased a ender 3 s1 (from microcenter with proof) with dual x screws, sprite extruder(single ribbon style) and filament run-out sensor; these devices now come standard in those configurations and are now set-up with tjc screens, instead of dwin or dacai and any new machines will not be compatible with the pro firmware... unless its a older units then this firmware may be done due to crealitys changes in hardware..
![Screenshot_20240909_125422_Firefox](https://github.com/user-attachments/assets/1c3f696c-8299-4bcf-b971-edd7fa3ac72c)
![20240909_130007](https://github.com/user-attachments/assets/2e924cae-63ba-4988-81e1-cbe602a1474b)
![IMG_8470](https://github.com/user-attachments/assets/0d07105f-c4af-4414-aaae-a62e86e73d83)
![IMG_8480](https://github.com/user-attachments/assets/605d264e-a10e-4aa1-a84c-5ee20720c95b)
![IMG_8473](https://github.com/user-attachments/assets/346fedb2-db82-4bf4-a053-a98050a41b52)

**Reply by mriscoc** [reply]
Check https://github.com/mriscoc/Ender3V2S1/discussions/745#discussioncomment-5437014

**Comment by mriscoc** [comment]
Please check the requirements: https://github.com/mriscoc/Ender3V2S1/wiki#requirements this firmware is compatible only with the listed boards.

**Comment by gitfve** [comment, marked as answer]
My Ender3.S1.PRO have the same board **CR4NS200141C13** with a **STM32F401RCT6** (ROM:256KB RAM:64KB).
It is running nice with fw: **"Ender3S1-F4-UBL-20230312.bin"**
Put you fw in the folder on your SDCard: **STM32F4_UPDATE**
url: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-install-the-firmware
Use a compatible display (i use the one from a Ender3V2):
url: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-update-the-display
